app/views/home/index.html.erb

```diff
@@ -1,4 +1,26 @@
 <h3>Puntstartr</h3>
+<div class="row">
+	<div class="col s5">
+  	<h5>Trending</h5>
+  </div>
+  <div class="col s5">
+    <h5>Nearly Funded</h5>
+  </div>  
+  <div class="col s2">
+    <div class="categories">
+      <div class="card-panel teal">
+        <span class="white-text">
+	     		<h5>Categories</h5>
+	       		<% @categories.each do |category| %>
```

Comment:
@stovermc Ah yeah, forgot to go through that file to make sure it looked good. I'll fix now.